docs/kerberos.md

```diff
@@ -146,13 +151,25 @@ You can also set the base64 encoded krb5.conf after install time:
 **Note** This setting `SPARK_MESOS_KRB5_CONF_BASE64` will overwrite/override any settings set with
 `SPARK_SECURITY_KERBEROS_KDC_HOSTNAME`, `SPARK_SECURITY_KERBEROS_KDC_PORT`, and `SPARK_SECURITY_KERBEROS_REALM`
 
+### Setting the Spark User
+
+By default, when Kerberos is enabled, Spark runs as the OS user coresponding to the
+Kerberos principal. For example, the principal "alice@LOCAL" would likely translate to "alice".
+Since "alice" would probably not be available as an OS user, it is recommended to
```

Comment:
Suggestion: "If it is known that "alice" is not available as an OS user, the Spark users should be specified as "root" or "nobody" instead:"

At the very least, I would not use the work "probably".

Reply:
We need to explain why there is this inconsistency.  Also it is possible to have an OS user eg. core on CoreOS and not on the executor image. The same issue appears.